app/styles/flex.scss

```diff
@@ -0,0 +1,43 @@
+.flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: start;
+}
```

Comment:
Note the default of column & start & start

IMO this is nice, because it matches a block layout.

Can easily be changed tho 🕺